notebooks/01_The_Ephys_Pipeline.ipynb

```diff
@@ -1,4983 +1,1447 @@
-{
```

Comment:
A word on Notebooks: Notebooks are not actual Python files, they're enormous HTML document trees.  (hence 6430 lines of change, most likely just from running the file). For that reason, they're virtually impossible to evaluate in the diff interface. 

Typically, I either discard all changes in notebooks prior to committing (if I ran the code without making any substantive changes), or stick a clear warning in the PR description field if I _have_ made substantive changes, so that the changes can be evaluated _in the notebook interface_ (which hides the enormous HTML mess).

It's not a _problem_ to commit non-substantive changes, but it's good policy not to (not doing so makes it easier to find commits where they _were_ substantively changed)

Reply:
Good to know. In this case I just didn't notice that the notebooks had also been committed until I started the PR, my bad! I'll pay more attention to what I'm committing in the future.